service/healthz/metrics.go

```diff
@@ -0,0 +1,29 @@
+package healthz
```

Comment:
I don't think these metrics are useful. We already provide metrics per route with microkit - see https://github.com/giantswarm/microkit/blob/b6bf741b5c34be4fff51d944f973318d8b078284/server/metrics.go